samples/embedded-auth-with-sdk/src/test/java/pages/Page.java

```diff
@@ -110,13 +121,43 @@ public String fetchCodeFromRegistrationEmail(String emailContent) {
         return matcher.find() ? matcher.group(1) : null;
     }
 
+    private void waitForNextTry() {
+        try {
+            Thread.sleep(getSleepDurationDuringVerificationCodeFetching());
+        } catch (InterruptedException e) {
+            e.printStackTrace();
+        }
+    }
+
     public String fetchCodeFromPasswordResetEmail(String emailContent) {
         Pattern pattern = Pattern.compile("Enter a code instead: (\\d{6})");
         Matcher matcher = pattern.matcher(emailContent);
         return matcher.find() ? matcher.group(1) : null;
     }
 
-    public void doNothing() {
-        try { Thread.sleep(500); } catch (InterruptedException e) { e.printStackTrace(); }
+    int getRetryCountDuringVerificationCodeFetching() {
+        int retry = 10;
```

Comment:
10 would probably introduce too much failures (flakiness) in ITs, I fear. Thoughts?

Reply:
Usually, it takes 2-3 seconds to get a code from sms/email. Using default value for OKTA_E2E_VERIFICATION_CODE_RETRY_COUNT=10 and OKTA_E2E_VERIFICATION_CODE_SLEEP_DURATION=2000 we have 20secs to get a verification code. Maybe it's better to increase OKTA_E2E_VERIFICATION_CODE_SLEEP_DURATION up to 3-4sec?